### ISSUE TITLE
Fails to parse "Quest Award Leveller - Knights of the Nine.esp"

### ISSUE THREAD

**Issue by WrinklyNinja** [WrinklyNinja/esplugin#8]
I've got no idea if this is a bug or if the plugin is invalid, but I'm sure I used to use this plugin back in the day. It's available for download here.

If the plugin isn't actually invalid, this issue is about fixing whatever is causing esplugin to think it is. If the plugin is invalid, then this issue is about improving esplugin's error reporting to include detail about what's invalid where.

**Comment by pStyl3** [comment]
For reference, here is a [screenshot](https://abload.de/img/knightswdjkr.jpg) of the plugin in question, and a link to the [modpage](http://www.nexusmods.com/oblivion/mods/5701).

**Comment by WrinklyNinja** [comment]
Oh, I forgot to actually include the link, thanks.

**Comment by WrinklyNinja** [comment]
It turns out that QAL - KOTN is corrupt, it has a few CELL records that aren't in a group. I've improved group parsing to validate that the group type is GRUP, which will help detect the issue correctly, and in the `descripting-parsing-errors` branch I've also added error strings to parsing errors. The latter is a breaking change, so I'm not going to merge it until there are more reasons to bump the major version number, but I'll close this issue now.
